lib/v2/threads/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:user?': ['ninboy'],
```

Comment:
```suggestion
    '/:user/:routeParams?': ['ninboy'],
```
https://github.com/DIYgod/RSSHub/blob/847f0bf087755d2c4b81b21aba445be5a40a269c/lib/v2/threads/router.js#L1-L3